packages/tools/examples/petCt/index.ts

```diff
@@ -698,7 +705,119 @@ async function setUpDisplay() {
   // Render the viewports
   renderingEngine.render();
 }
+addButtonToToolbar({
+  title: 'toggle volume3d',
+  onClick: async () => {
+    const ptImageIds = await getPtImageIds();
+    // Define a volume in memory
+    const volume = await volumeLoader.createAndCacheVolume(ptVolumeId, {
+      imageIds: ptImageIds,
+    });
+    volume.load();
```

Comment:
you don't need to create another volume for this, the same volume can be used in a different viewport type